src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/SimpleTypeInfos.cs

```diff
@@ -261,21 +261,21 @@ public override void WriteData(PropertyValue value)
         }
     }
 
+#nullable enable
```

Comment:
Hmm, I thought everything in corelib already had nullable enabled on it.

Reply:
Ah, maybe it does in the project file, I didn't check.

Reply:
It does :)
https://github.com/dotnet/runtime/blob/b8db68c52d817b2cb0bff30c0f43eb47901c5d2f/src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj#L47
https://github.com/dotnet/runtime/blob/b8db68c52d817b2cb0bff30c0f43eb47901c5d2f/src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj#L40

Reply:
(We've only left #nullable enable in files that are shared with other projects not yet nullable enabled, or where we've forgotten to remove it :)